tests/test_subclass.py

```diff
@@ -12,10 +12,9 @@ class LabeledVector(Vector2):
     """Subclass of Vector2 that defines an additional attribute."""
     label: str
 
-    def __new__(cls, x, y, label):
-        self = super().__new__(cls, x, y)
+    def __init__(self, x, y, label):
+        Vector2.__init__(self, x, y)
```

Comment:
So, calling the base class explicitly like this is actually an older (and less accepted now) pattern in Python. What was the problem you were trying to solve?

Reply:
Yeah, this come from #146 where I had initially used `super()`, but messed it up and ended up calling directly so it would work.  @astronouth7303 already commented on that and it already got fixed.

Reply:
See https://github.com/ppb/ppb-vector/pull/144#discussion_r275822548